header-only/layout-builder/awkward/LayoutBuilder.h

```diff
@@ -1088,7 +1088,6 @@ namespace awkward {
     /// @class Indexed
     ///
     /// @brief Builds an IndexedArray which consists of an `index` buffer.
-    /// The negative values in the index are interpreted as missing.
     ///
     /// The index values can be 64-bit signed integers `int64`, 32-bit signed
     /// integers `int32`.
```

Comment:
```suggestion
    /// The index values can be 64-bit signed integers `int64`, 32-bit signed
    /// integers `int32`.
```
The valid types are - 64-bit signed integers `int64`, 32-bit signed integers `int32` or 32-bit unsigned integers `uint32`.